docker-compose.yml

```diff
@@ -13,13 +13,12 @@ services:
       postgres:
         condition: service_healthy
     volumes:
-      - ./app:/app
-      - mellow-bike-map-node-modules:/app/node_modules
+      - .:/app
+      - mellow-bike-map-node-modules:/app/app/node_modules
     environment:
       DJANGO_SECRET_KEY: reallysupersecret
       DJANGO_MANAGEPY_MIGRATE: "on"
-    entrypoint: /app/docker-entrypoint.sh
-    command: python manage.py runserver 0.0.0.0:8000
+    command: /app/app/docker-entrypoint.sh python manage.py runserver 0.0.0.0:8000
```

Comment:
Note for posterity: Something weird was happening with `docker-entrypoint.sh` where it couldn't actually read in command overrides as arguments when specifying a new command via the CLI. I never really figured it out, but refactoring things such that `docker-entrypoint.sh` is used in the command rather than the entrypoint fixed the problem, and more closely matches the way I work in development anyway (I don't want to run migrations literally every time I'm running code on the `app` image).